package-lock.json

```diff
@@ -2832,9 +2832,73 @@
 			"dev": true,
 			"requires": {
 				"babel-eslint": "^8.0.3",
-				"eslint-plugin-jsx-a11y": "6.0.2",
-				"eslint-plugin-react": "7.7.0",
+				"eslint-plugin-jsx-a11y": "^6.2.1",
+				"eslint-plugin-react": "^7.12.4",
 				"requireindex": "^1.2.0"
+			},
+			"dependencies": {
```

Comment:
Handling `package-lock.json` with local packages is hard. Ideally `eslint-plugin-jsx-a11y` and `eslint-plugin-react` are installed in the global `node_modules` folder. I think removing manually this `dependencies` section and running `npm install` should move them again to the root.

Reply:
As expected Eslint fails because can't find those packages:

https://travis-ci.org/WordPress/gutenberg/jobs/498058861#L689

> ESLint couldn't find the plugin "eslint-plugin-react". This can happen for a couple different reasons:

Reply:
Thanks for your help! Travis seems to be working fine now. There are just some ESLint errors that need to be addressed.